src/test/teams.test.ts

```diff
@@ -1341,7 +1341,104 @@ describe('Teams resource', () => {
       await MongoDB.Teams.removeByTeamId(secondTeam.teamid),
       await MongoDB.Teams.removeByTeamId(thirdTeam.teamid)
     });
+  });
+
+  describe('DELETE team when no members', () => {
+
+    let attendee: IAttendee;
+    let team: ITeam;
+    let statusCode: number;
+    let contentType: string;
+    let body: string;
+
+    before(async () => {
+      attendee = await MongoDB.Attendees.insertRandomAttendee();
+      team = await MongoDB.Teams.insertRandomTeam([], 'ABCD');
+
+      const res = await api.delete(`/teams/${team.teamid}`)
+        .auth(attendee.attendeeid, ApiServer.HackbotPassword)
+        .type('application/vnd.api+json')
+        .send()
+        .end();
+
+      statusCode = res.status;
+      contentType = res.header['content-type'];
+      body = res.text;
+    });
+
+    it('should respond with status code 204 No Content', () => {
+      assert.strictEqual(statusCode, 204);
+    });
+
+    it('should not return a content-type', () => {
+      assert.strictEqual(contentType, undefined);
+    });
+
+    it('should not return a response body', () => {
+      assert.strictEqual(body, '');
+    });
+
+    it('should delete the team', async () => {
+      const result = await MongoDB.Teams.findbyTeamId(team.teamid);
+      assert.strictEqual(result, null);
+    });
+
+    after(async () => {
+      await MongoDB.Attendees.removeByAttendeeId(attendee.attendeeid);

```

Comment:
I would probably just `return` rather than mark this as `async` and `await`. But it doesn't really matter.